### DISCUSSION TITLE
Cannot start a container because mkdir reports file exists

### DISCUSSION THREAD

**Discussion by s-gude** [moby/moby#46151] [Q&A]
I am currently facing an issue while working with Docker, and I could use some assistance from the community.
I am new to docker, I want to start a simple container with "docker start dashboard" .

The following error occurs when I try to start a container named "dashboard" using Docker. I have searched for a solution online and tried a few troubleshooting steps, but unfortunately, I haven't been able to resolve it yet.

If anyone has encountered a similar issue before or has insights into the root cause and possible solutions, I would greatly appreciate your help. Your expertise and suggestions will be valuable in getting past this roadblock.

```
`s-gude@ip-100-84-80-21:~# docker start dashboard`
**Error response from daemon: mkdir /run/containerd/io.containerd.runtime.v1.linux/moby/f16e4106fd1f9b943a60e4e5ece7ac518ccd209d602e0a6f448c07a707eb7dec: file exists: unknown
Error: failed to start containers: dashboard**
```
**Docker version:**
`s-gude@ip-100-84-80-21:~# docker version
Client: Docker Engine - Community
 Version:           19.03.6
 API version:       1.40
 Go version:        go1.12.16
 Git commit:        369ce74a3c
 Built:             Thu Feb 13 01:27:49 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.6
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.16
  Git commit:       369ce74a3c
  Built:            Thu Feb 13 01:26:21 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683`

**Docker Info:**
`s-gude@ip-100-84-80-21:~# docker info
Client:
 Debug Mode: false

Server:
 Containers: 12
  Running: 0
  Paused: 0
  Stopped: 12
 Images: 8
 Server Version: 19.03.6
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: b34a5c8af56e510852c35414db4c1f4fa6172339
 runc version: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-1032-aws
 Operating System: Ubuntu 18.04.4 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 456.1MiB
 Name: ip-100-64-60-21
 ID: NGOB:QBID:TMOW:EOGV:USU6:N6AA:NVIX:LR33:53UD:MHBE:NOWS:H2BQ
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support`

Thank you in advance for any assistance you can provide. Please feel free to leave a comment or reach out to me directly if you have any insights or ideas.

**Comment by corhere** [comment]
> Server Version: 19.03.6

That version is four years out of date and has reached end of life. Try upgrading to a maintained version.
